### ISSUE TITLE
Several testfiles have old version numbers

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5805]
## Issue overview

Happened to notice version warnings while testing all of the new (and old) DOATo\* testfiles.  Searching current develop finds files listed below with old version numbers.  Maybe some need to be transitioned up?  Most probably just need the version number changed to 8.6, but should test them all.  

Also, would be good to run all the testfiles through transition with the MakingPretty option set to clean up old field names and fix formatting.
### Checklist

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
- [x] Ticket added to Pivotal for defect (development team task)
- [x] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)
### Details

```
\testfiles\AdvancedOutput\ExerciseOutput1.idf(7):   VERSION,8.3;
\testfiles\AdvancedOutput\ExerciseOutput1A-Solution.idf(7):   VERSION,8.3;
\testfiles\5ZoneFanCoilDOAS_HumidifierOnOASystem.idf(108): Version,8.4;
\testfiles\CoolingCoilFreezingPrevention.idf(105):   Version,8.4;
\testfiles\DOAToWaterToAirHPInlet.idf(109):     8.4;                     !- Version Identifier
\testfiles\DOAToWaterToAirHPSupply.idf(109):     8.4;                     !- Version Identifier
\testfiles\ZoneCoupledGroundHTBasement.idf(65):     8.4; !- Version Identifier
\testfiles\ZoneCoupledGroundHTSlabInGrade.idf(52):     8.4; !- Version Identifier
\testfiles\ZoneCoupledGroundHTSlabOnGrade.idf(52):     8.4; !- Version Identifier
\testfiles\BasicsFiles\AdultEducationCenter.idf(87):   VERSION,8.4;
\testfiles\BasicsFiles\Exercise1A.idf(17):   VERSION,8.4;
\testfiles\BasicsFiles\Exercise1B-Solution.idf(80):   VERSION,8.4;
\testfiles\BasicsFiles\Exercise1C-Solution.idf(126):   VERSION,8.4;
\testfiles\BasicsFiles\Exercise1D-Solution.idf(149):   VERSION,8.4;
\testfiles\BasicsFiles\Exercise2.idf(34):   VERSION,8.4;
\testfiles\BasicsFiles\Exercise2A-Solution.idf(131):   VERSION,8.4;
\testfiles\BasicsFiles\Exercise2B-Solution.idf(410):   VERSION,8.4;
\testfiles\BasicsFiles\Exercise2C-Solution.idf(430):   VERSION,8.4;
\testfiles\MovableIntInsulationLights.idf(15):   Version,8.5;
\testfiles\MovableIntInsulationLightsLowE.idf(15):   Version,8.5;
\testfiles\VariableRefrigerantFlow_FluidTCtrl_HR_5Zone.idf(99):   Version,8.5;
```

**Comment by mjwitte** [comment]
@mitchute If you're willing, it would be good to test transition on all of the v8.5 testfiles (of course they all need to be version 8.5 first) to see if they all run after transition?

**Comment by mitchute** [comment]
@mjwitte I'm planning on it. Thanks!

**Comment by mjwitte** [comment]
Closed via #5859
